controller/handler_edge_ctrl/heartbeat.go

```diff
@@ -38,17 +39,35 @@ func (h *sessionHeartbeatHandler) ContentType() int32 {
 	return env.ApiSessionHeartbeatType
 }
 
-func (h *sessionHeartbeatHandler) HandleReceive(msg *channel2.Message, _ channel2.Channel) {
+func (h *sessionHeartbeatHandler) HandleReceive(msg *channel2.Message, ch channel2.Channel) {
```

Comment:
@plorenz I added some heartbeat invalid api token logic here. You did your review before I pushed this.